pegasus/sites.v3/code.org/public/teacher-dashboard/section_assessments.haml

```diff
@@ -85,7 +85,7 @@ content-type: text/ng-template
   &nbsp;
 
 :ruby
-  levels_filtered = "levels | orderBy:order | filter:(stageFilter && {stage: stageFilter}):true"
+  levels_filtered = "levels | orderBy:orderSingles | filter:(stageFilterSingles && {stage: stageFilterSingles}):true"
```

Comment:
maybe also change the name of the other order and stageFilter vars? Also, do we have a consistent naming scheme for what we call these things (since "Singles" seems like new vocabulary...).